.github/workflows/sdist.yml

```diff
@@ -0,0 +1,118 @@
+name: Sdist-installation
+
+on:
+  workflow_dispatch:
+    inputs:
+      python_version:
+        required: true
+        type: string
+      ref:
+        required: false
+        type: string
+      check_run_id:
+        required: false
+        type: string
+      pr_repo:
+        required: false
+        type: string
+  push:
+    branches: [devel, main]
+
+env:
+  COMMIT: ${{ inputs.ref || github.event.ref }}
+  PEM: ${{ secrets.BOT_PEM }}
+  GITHUB_RUN_ID: ${{ github.run_id }}
+  GITHUB_CHECK_RUN_ID: ${{ inputs.check_run_id }}
+  PR_REPO: ${{ inputs.pr_repo || github.repository }}
+
+jobs:
+  Python_version_picker:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'push' || github.repository == 'pyccel/pyccel'
+    outputs:
+      python_version: ${{ steps.set-python_version.outputs.python_version }}
+    steps:
+    - if: github.event_name == 'push'
+      run: |
+        echo "version=3.10" >> $GITHUB_ENV
+      shell: bash
+    - id: dispatch-matrix
+      if: github.event_name == 'workflow_dispatch'
+      run: |
+        echo "version=${{ inputs.python_version }}" >> $GITHUB_ENV
+      shell: bash
+    - id: set-python_version
+      run: |
+        echo "python_version=$version" >> $GITHUB_OUTPUT
+      shell: bash
+
+  Wheel-installation:
+    runs-on: ubuntu-latest
+    name: Deploy tests
```

Comment:
Why calling this job "Deploy tests"?

Reply:
Because it tests if you can create the sdist object that is created during the deployment